### DISCUSSION TITLE
Easier install on windows from zip through scoop

### DISCUSSION THREAD

**Discussion by boltomli** [ballerina-platform/ballerina-lang#37096] [General]
To make it easier to install from zip, after installing [scoop](https://scoop.sh/):

```
scoop bucket add mywin https://github.com/boltomli/MyWinScripts
scoop install ballerina
```

I didn't bother to suggest jdk/jre, which most people here should have installed their favourite dist.

**Comment by pcnfernando** [comment]
looping in @keizer619

**Comment by keizer619** [comment]
I went into MyWinScripts but unfortunately i didn't find any ballerina references there
